src/stream/src/executor/wrapper/trace.rs

```diff
@@ -38,14 +39,16 @@ pub async fn trace(
 
     let span_name = pretty_identity(&info.identity, actor_id, executor_id);
 
+    let is_sink_or_mv = info.identity.contains("Materialize") || info.identity.contains("Sink");
```

Comment:
This seems fragile but I'm unsure if there's a better way.

Reply:
let me open an issue to track this, a quick fix for Kaito

Reply:
+1. It seems a little bit hacky here.